src/popen.rs

```diff
@@ -607,7 +607,6 @@ mod os {
     use std::ffi::OsString;
     use std::time::{Duration, Instant};
 
-    use super::ChildState::*;
```

Comment:
Oh, I see.  I guess this changed in Rust 1.15 because of rust-lang/rust#35120.  I have removed this change from the PR.